json_serializable/lib/src/decode_helper.dart

```diff
@@ -317,27 +317,18 @@ _ConstructorData _writeConstructorInvocation(
       '$className'
       '${genericClassArguments(classElement, false)}'
       '$constructorExtra(',
-    );
-  if (constructorArguments.isNotEmpty) {
-    buffer
-      ..writeln()
-      ..writeAll(constructorArguments.map((paramElement) {
-        final content =
-            deserializeForField(paramElement.name, ctorParam: paramElement);
-        return '      $content,\n';
-      }));
-  }
-  if (namedConstructorArguments.isNotEmpty) {
-    buffer
-      ..writeln()
-      ..writeAll(namedConstructorArguments.map((paramElement) {
-        final value =
-            deserializeForField(paramElement.name, ctorParam: paramElement);
-        return '      ${paramElement.name}: $value,\n';
-      }));
-  }
-
-  buffer.write(')');
+    )
+    ..writeAll(constructorArguments.map((paramElement) {
```

Comment:
The only real change here is the removal of the extra `..writeln()` but figured I'd do more general cleanup, too!